include/grpc/grpc.h

```diff
@@ -162,6 +162,29 @@ GRPCAPI void grpc_completion_queue_thread_local_cache_init(
 GRPCAPI int grpc_completion_queue_thread_local_cache_flush(
     grpc_completion_queue* cq, void** tag, int* ok);
 
+/************* CHANNELZ API *************/
```

Comment:
Maybe move all of this down to the bottom, so that it's clear that this comment only applies to the two methods.